constraint/pkg/apis/templates/yaml_constant.go

```diff
@@ -310,4 +310,58 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
+---
```

Comment:
This shouldn't be making its way in here.  It has the potential to break the logic in `pkg/apis/templates/crd_schema.go`.

Easiest fix I can think of is to have `controller-gen` put the CRDs each into their own files, so that we can grab the correct file in the Makefile.  (See the `CRD_SOURCE_FILE` variable in the [Makefile](https://github.com/open-policy-agent/frameworks/blob/master/constraint/Makefile#L66))

Reply:
looks like we were just copying crds.yaml to this. Changed it to `config/crds/templates.gatekeeper.sh_constrainttemplates.yaml`

Reply:
looks like there is a diff in preserveUnknownFields. Do you know why is there a difference between `crds.yaml` and `config/crds/templates.gatekeeper.sh_constrainttemplates.yaml`?

Reply:
Is what you're referencing showing up in this PR?  I don't see `preserveUnknownFields` anywhere in this diff page.